source/CONTRIBUTING.md

```diff
@@ -74,8 +74,8 @@ The answers are posted on [discuss.emberjs.com](https://discuss.emberjs.com/) an
 
 ### Writing Style Guide
 
-- Use Title Case for section headings, when in doubt use https://titlecase.com/.
```

Comment:
These lines seemed to be causing Travis to fail.

Reply:
Awesome, thank you for the fix :sparkles: